<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/kword.png</normaloff>:/icons/kword.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelDeclaration">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QTabWidget" name="tab_5">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>Documentation</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="cs8ProgramHeaderView" name="widgetDocumentation" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Parameters</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="cs8ProgramDataView" name="tableViewPars"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Variables</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="cs8ProgramDataView" name="tableViewVars"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_4">
               <attribute name="title">
                <string>Referenced Global Variables</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="cs8ProgramDataView" name="tableViewReferencedGlobalVaribales"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="formVariableDetailEditor" name="detailEditor" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="CodeEditor" name="plainTextEditCode">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="menuRecent"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionEdit_Copy_Right_Message"/>
    <addaction name="actionEdit_Project_Documentation"/>
    <addaction name="actionEdit_Main_Page"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_tags_for_undocumented_symbols"/>
    <addaction name="actionInclude_documentation_of_Libraries"/>
    <addaction name="separator"/>
    <addaction name="actionBuild_Documentation"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionApply_project_data_to_all_sub_projects"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
   <addaction name="menuTools_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>190</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListView" name="listViewProgams">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="cs8ProgramDataView" name="globalDataView">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEdit_Copy_Right_Message">
   <property name="text">
    <string>Edit Copy Right Message</string>
   </property>
  </action>
  <action name="actionAdd_tags_for_undocumented_symbols">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add tags for undocumented symbols</string>
   </property>
  </action>
  <action name="actionApply_project_data_to_all_sub_projects">
   <property name="text">
    <string>Apply project data to all sub projects</string>
   </property>
  </action>
  <action name="actionEdit_Project_Documentation">
   <property name="text">
    <string>Edit Project Documentation</string>
   </property>
  </action>
  <action name="actionEdit_Main_Page">
   <property name="text">
    <string>Edit Main Page</string>
   </property>
  </action>
  <action name="actionBuild_Documentation">
   <property name="text">
    <string>Build Documentation</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionInclude_documentation_of_Libraries">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Include documentation of Libraries</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cs8ProgramDataView</class>
   <extends>QTableView</extends>
   <header>cs8programdataview.h</header>
  </customwidget>
  <customwidget>
   <class>formVariableDetailEditor</class>
   <extends>QWidget</extends>
   <header>formvariabledetaileditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cs8ProgramHeaderView</class>
   <extends>QWidget</extends>
   <header>cs8programheaderview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editParameterDetail(QModelIndex)</slot>
  <slot>editProjectDocumenation()</slot>
 </slots>
</ui>
